### DISCUSSION TITLE
Account with this email already exists

### DISCUSSION THREAD

**Discussion by bendem** [dani-garcia/vaultwarden#1883] [Q&A]
### Subject of the issue
After inviting a new user, they cannot create an account. They are told their email is already in use.

### Deployment environment

* vaultwarden version: 1.22.2

* Install method: docker image

* Clients used: browser

* Reverse proxy and version: httpd in docker

* MySQL/MariaDB or PostgreSQL version: postgresql 10

* Other relevant details:
Using `SIGNUPS_DOMAINS_WHITELIST` and `SIGNUPS_VERIFY`.

### Steps to reproduce
* Invite a non-existing user to organisations
* Wait for them to create account

### Expected behaviour
* Account is created, user joins organisation

### Actual behaviour
* User is told `Account with this email already exists`

### Troubleshooting data
```
[2021-08-04 10:58:39.930][response][INFO] POST /api/organizations/<org_id>/users/invite (send_invite) => 200 OK
...
[2021-08-04 11:06:19.456][request][INFO] POST /api/accounts/register
[2021-08-04 11:06:19.468][vaultwarden::api::core::accounts][ERROR] Account with this email already exists
```

**Comment by BlackDex** [comment]
Works for me.
The user probably didn't used the correct link which includes the token which is validated during the account creation.
Please resend the invite and let the user try again.

**Reply by bendem** [reply]
Ok, I can understand how it's going wrong: I've invited the user to multiple organisations. Each new invite invalidates the link in the previous mail or something.
